cookbooks/continental_compression_with_imposed_faults/doc/continental_compression_with_imposed_faults.md

```diff
@@ -0,0 +1,56 @@
+(sec:cookbooks:Continental-Compression-with-imposed-faults)=
+# Continental Compression with Pre-Existing Fault inheritance
+
+*This section was contributed by Prajakta Mohite and John Naliboff.*
+
+Recent numerical modeling investigations have demonstrated the influence of extensional phases of deformation on the subsequent evolution of continental collision zones (eg., {cite}`Vasey:etal:2024`,{cite}`Zwaan:etal:2025`). Motivated by these studies and a wide range of observations that highlight fault reactivation processes during the Wilson cycle, this cookbook implements a 2D visco-plastic model of continental compression that incorporates pre-existing fault zones into the initial conditions.
+
+The cookbook builds directly on components of the {ref}`sec:cookbooks:crustal-deformation` cookbook, and the primary goal here is to:
+1. Highlight techniques for imposing fault zones in the initial conditions of a lithospheric deformation simulation and
+2. Demonstrate their effect on the evolution of deformation patterns.
+
+# Model Design
+The model domain spans 400 km x 400 km and uses adaptive refinement to resolve deformation patterns in the regions where faults are imposed at the onset of deformation. The initial thermal structure follows a conductive, continental-style geothermal through the lithosphere and an initial adiabatic profile in the asthenosphere. The governing equations follow the extended Boussinesq approximation, which includes both adiabatic and shear heating.
+
+Deformation is driven by horizontal velocity applied at the model sides (2 mm/yr), which are balanced by outflow at the model base. A free surface allows topography to develop through time, which is diffused at each time step to approximate landscape evolution processes and stabilize both linear and nonlinear solver behavior.
```

Comment:
```suggestion
Deformation is driven by horizontal velocity applied at the model sides (1 mm/yr), which are balanced by outflow at the model base (2 mm/yr). A free surface allows topography to develop through time, which is diffused at each time step to approximate landscape evolution processes and stabilize both linear and nonlinear solver behavior.
```
I think it's more clear to say that you apply 1 mm/yr to the sides, which balances the 2 mm/yr you apply to the bottom